fixtures/view_helper.rb

```diff
@@ -19,4 +21,18 @@ def svg_view(&block)
 			Class.new(Phlex::SVG, &block)
 		end
 	end
+
+	module Utils
+		def capture_stderr(&block)
+			captured_stream = StringIO.new
+			original_stream = $stderr
+			$stderr = captured_stream
+
+			block.call
+
+			captured_stream.string.strip
+		ensure
+			$stderr = original_stream
+		end
```

Comment:
I added this as a module because I think it can be useful for other deprecation tests in the future.